src/customKeywords/files.js

```diff
@@ -0,0 +1,46 @@
+var Ajv = require('ajv');
+const fileNameField = 'fieldname';
+
+function missingElements(superSet, subset) {
+    let missingElements = [];
+    subset.every(function (subsetElement) {
+        if (!superSet.includes(subsetElement)) {
+            missingElements.push(subsetElement);
+        }
+        return true;
+    });
+
+    return missingElements;
+};
+
+module.exports = {
+    validate: function filesValidation(schema, data) {
+        filesValidation.errors = [];
+        const dataFileName = data.map((element) => { return element[fileNameField] });
+        let missingFiles = missingElements(dataFileName, schema.required);
+        if (missingFiles.length > 0) {
+            filesValidation.errors.push(new Ajv.ValidationError({
+                keyword: 'files',
+                message: 'Missing required files: ' + schema.required.toString(),
```

Comment:
Is this message going to be correct? I presume it will list all required files, not just the missing ones?

Reply:
good point, fixed that one